templates/taxi/driver_detail.html

```diff
@@ -22,5 +23,6 @@ <h4>Cars</h4>
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+  <a href="{% url 'taxi:driver-delete' pk=driver.id %}"><button class="btn btn-primary">Delete</button></a>
```

Comment:
Consider using `driver.id` instead of `pk=driver.id` in the URL tag to maintain consistency with other parts of the template and avoid potential confusion if `pk` is not defined in the context.